src/viewer.c

```diff
@@ -57,12 +55,13 @@ static const char* scale_names[] = {
 
 /** Viewer context. */
 struct viewer {
-    ssize_t img_x, img_y; ///< Top left corner of the image
-    size_t frame;         ///< Index of the current frame
-    argb_t image_bkg;     ///< Image background mode/color
-    argb_t window_bkg;    ///< Window background mode/color
-    bool antialiasing;    ///< Anti-aliasing mode on/off
-    bool fixed;           ///< Fix image position
+    ssize_t img_x, img_y;  ///< Top left corner of the image
+    size_t frame;          ///< Index of the current frame
+    argb_t image_bkg;      ///< Image background color
+    argb_t window_bkg;     ///< Window background color
+    bool image_bkg_grid;   ///< Whether image background is grid
```

Comment:
Now we can set  both values: color and grid at same time, but they are mutually exclusive options.

Reply:
I figured this was the most effective way to convey the background preferences, now that every possible value of `argb_t` is used, even if the color becomes redundant in the grid case. Regardless, the acceptance of arbitrary alpha values requires the type's size to increase since none of the existing values of `argb_t` are available for non-color interpretation.

Could be replaced by a 64 bit type or another struct, which if preferable I would be happy to add to the PR.